<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>555</width>
      <height>302</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1" colspan="2">
      <widget class="QSpinBox" name="Posicion">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="3">
      <widget class="QLabel" name="label">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>TareaExtraclase Datos 2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6" rowspan="7">
      <widget class="QGraphicsView" name="lienzo">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QLabel" name="label_2">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Escoja su estructura de Datos</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QComboBox" name="Estructura">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>pos</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Lists</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="DeleteB">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>DeleteBegin</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QPushButton" name="DeleteE">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>DeleteEnd</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="DeleteP">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>DeleteByPos</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="AddB">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>AddBegin</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="2">
      <widget class="QPushButton" name="AddE">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>AddEnd</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QPushButton" name="AddP">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>AddByPos</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QPushButton" name="SetV">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>SetValue</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="2" colspan="2">
      <widget class="QPushButton" name="GetV">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>GetValue</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QPushButton" name="PrintL">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Print</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>BinaryTree</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QPushButton" name="AddT">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="2" colspan="2">
      <widget class="QPushButton" name="DeleteT">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>DeleteNode</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QSpinBox" name="valor">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="4">
      <widget class="QPushButton" name="PrintT">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Print</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="2" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Go</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_4">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>value</string>
       </property>
      </widget>
     </item>
     <item row="8" column="6">
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="6">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Largo Lista</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
